sci-libs/gdal/gdal-2.0.2-r1.ebuild

```diff
@@ -0,0 +1,321 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+WANT_AUTOCONF="2.5"
+
+GENTOO_DEPEND_ON_PERL="no"
+PYTHON_COMPAT=( python2_7 python3_{3,4} )
+DISTUTILS_OPTIONAL=1
+
+inherit autotools eutils libtool perl-module distutils-r1 python-r1 toolchain-funcs java-pkg-opt-2
```

Comment:
This ebuild is not using distutils-r1 at all… (I know you haven't touched it but it's a major problem and since you are already touching Python stuffs, it would be nice to have this fixed)